src/services/request.service.js

```diff
@@ -96,11 +95,29 @@ const getManagerRequest = async (userId) => {
   return requests;
 };
 
+const updateRequestStatus = async (requestId, status) => {
+  if (status === 'approved' || status === 'declined') {
+    const request = await db.request.update({ status }, {
+      where: {
+        id: requestId,
+        status: 'open'
+      },
+    });
+    if (request[0] === 0) {
```

Comment:
It is hard to understand the `condition` in the if statement. The condition can be given a better name. A readable variable can take up `request[0] === 0`.

Reply:
Thanks, working on it